tests/test_http.py

```diff
@@ -62,7 +64,10 @@ def setUp(self):
 
         self.apns_pushkin_snotif = MagicMock()
         for key, value in self.sygnal.pushkins.items():
-            value._send_notification = self.apns_pushkin_snotif
+            assert isinstance(value, ApnsPushkin)
+            # type safety: ignore is used here due to mypy not handling monkeypatching,
+            # see https://github.com/python/mypy/issues/2427
+            value._send_notification = self.apns_pushkin_snotif  # type: ignore
```

Comment:
Where possible, prefer `#type: ignore[code]` where `code` is one of mypy's error codes. See https://mypy.readthedocs.io/en/stable/error_codes.html and the two pages following it for gory details.

(but easier to remove the comment and see what the error code you get in the terminal is.)